google-cloud-storage/src/main/java/com/google/cloud/storage/multipartupload/model/CompleteMultipartUploadRequest.java

```diff
@@ -84,6 +86,17 @@ public CompletedMultipartUpload multipartUpload() {
     return multipartUpload;
   }
 
+  /**
+   * Returns the user-project.
+   *
+   * @return the user-project.
+   * @since 2.62 This new api is in preview and is subject to breaking changes.
```

Comment:
This release will be 2.61.0 https://github.com/googleapis/java-storage/pull/3392

Reply:
Done